### ISSUE TITLE
Dark mode: Toggle button's background has insufficient contrast against its adjacent color

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#841]
**Describe the bug**
- The background color of the toggle buttons in settings have insufficient contrast against their adjacent color.
- The background color of the inner circle inside the toggle button also has insufficient contrast against its adjacent color (which is the background color of the toggle)

**Toggle button's background color:** #0078D6
**Adjacent color:** #2B3034 (also the background color of the inner circle)

**To Reproduce**
1. Make sure color setting for default app mode in windows is set to 'Dark'
2. Launch AI-Windows
3. Go to settings -> Application
4. User Color Contrast Analyser tool to pick the toggle's background color and its adjacent color
5. Observe that the contrast ratio is less than the desired ratio of 3:1 for non-text components

**Expected behavior**
The toggle buttons' background color should have sufficient contrast of 3:1 against the adjacent color

**Actual behavior**
Non-text contrast ratio is less than the expected 3:1

**Screenshots or .GIF**
![1](https://user-images.githubusercontent.com/22068434/87972262-7753f300-ca7b-11ea-849b-d42965b0179b.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by adarshrema** [comment]
Needs investigation by PM and design.

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in the release at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1382.01. Please regress/close at your convenience.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1383.01
